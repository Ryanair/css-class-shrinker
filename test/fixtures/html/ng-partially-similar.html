<div class="aaaooo"></div>
<div class="eee">
  <h1 ng-class="{'iii': true, 'aaa-ooo-uuu': false}"></h1>
  <button class="ooo"></button>
  <div ng-class="{'uuu': true, 'qqq-eee': false}"></div>
</div>
