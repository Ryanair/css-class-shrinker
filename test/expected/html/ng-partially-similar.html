<div class="aaaooo"></div>
<div class="eee">
  <h1 ng-class="{'fsG': true, 'aaa-ooo-uuu': false}"></h1>
  <button class="ooo"></button>
  <div ng-class="{'gDs': true, 'qqq-eee': false}"></div>
</div>
